fastparquet/api.py

```diff
@@ -122,36 +122,35 @@ def row_group_filename(self, rg):
         else:
             return self.fn
 
-
     def read_row_group_file(self, rg, columns, categories, index=None,
-                            assign=None):
+                            assign=None, mr_times=[]):
```

Comment:
To me this is a timestamp (like January 1st, 2001) rather than an interval (like one year and fifteen seconds).  Is this encoding common?  Are there other competing timestamp96 standards?